tests/test_interpolatedimage.py

```diff
@@ -1647,6 +1713,129 @@ def test_quintic_glagn():
 
         gsobj.drawImage(method='phot', image=image, add_to_image=True)
 
+def test_depixelize():
+    # True, non-II profile.  Something not too symmetric.
```

Comment:
This is not too symmetric, but it is a quite simple profile.  In practice, people will likely want to use this for PSF models, and I wonder if it would be valuable to have a test using say Kolmogorov convolved with Airy or OpticalPSF?  OpticalPSF, especially, has some features that I could imagine pose challenges for the deconvolution.  I realize this could take longer so it could be a specialized test that is not run routinely but is run before new releases, or something like that.  (like the `do_slow_tests` option in `test_optics.py`)

Reply:
I switched to something more complicated:
```
true_prof = galsim.Convolve(
                    galsim.Kolmogorov(fwhm=1.1, flux=10).shear(g1=0.1, g2=0.2),
                    galsim.OpticalPSF(lam_over_diam=0.6, obscuration=0.4, nstruts=4,
                                      defocus=0.15, astig1=0.13, astig2=-0.14,
                                      coma1=-0.06, trefoil1=-0.08))
```
<img width="126" alt="Screen Shot 2022-06-16 at 5 01 50 PM" src="https://user-images.githubusercontent.com/623887/174163039-c29a6fdc-dda0-41d2-8f9b-cc5798e60267.png">
And the existing tests still passed.  TBH, I suspect noise is a harder thing for it to deal with, since it doesn't represent a true profile convolved by a pixel.  But I haven't investigated how low noise is required for this to end up sensible.